docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -14,21 +14,21 @@ In addition to a working laptop or desktop computer with a decent internet conne
 
 #### Step 1. Navigate to [app.aragon.one](http://app.aragon.one) in your web browser and click “Create a new organization”
 
-![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/welcome.png)
+![](https://github.com/aragon/aragon-wiki/blob/john-light-patch-1/docs/images/user_guide/welcome.png)
```

Comment:
> no clear reason for the inconsistency here?

The reason is because I figured out the right way to do it with the second batch of images and didn't feel like messing with the first batch of images :p the breakage occurred when I tried to be clever and change the image path, but could figure out how with the folder the first set of images were in, so tried to change back to using the full URL but looks like I grabbed the URL from the wrong branch.

Not a good reason, but that's the reason.

Reply:
Okay, still waiting to see the changes made to the image paths, and should always use relative paths instead of external urls.

Reply:
resolved

Reply:
@Smokyish on second look, now my images in the Multisig DAO section aren't showing up even though I entered the relative path. Can you take a look and see if you can help me diagnose why they aren't showing up?